docs/adr/0015-security-fixes-non-wmde-software.md

```diff
@@ -0,0 +1,74 @@
+# 15) Including and Publishing Security fixes {#adr_0015}
+
+Date: 2021-05-03
+
+## Status
+
+proposed
+
+## Context
+
+WMDE needs to decide on a process for including security fixes of non-WMDE maintained software (e.g. Mediawiki, Elasticearch, WDQS) as well as WMDE maintained software (Wikibase, WDQS-frontend) and how to publish security fix releases of Wikibase suite in such cases.
+
+Security fixes will fall under the maintenance category and will most likely be worked on by campsite, however as the running of the pipeline and publishing of docker images requires special privileges it could make sense that this step is tasked to a certain group of people or that all engineers of WMDE are given the rights to publish releases.
+
+## List of steps reporting, fixing and releasing security updates
```

Comment:
Steps 1-3 seem to only be relevant for WMDE maintained software (plus possibly reporting sec bugs in WMF's stuff). It kind of is cleared in the paragraphs above, but maybe it would still make sense to make it super clear that this section is about WMDE's stuff only?

Reply:
done